plgrid/plgdata_methods.py

```diff
@@ -0,0 +1,52 @@
+import requests
+
+import tempfile
+
+from pathlib import Path
+
+from plgrid.string_templates import plgdata_get_url, plgdata_list_url
+
+from pymchelper.input_output import fromfile
+
+from yaptide.celery.tasks import pymchelper_output_to_json
+
+http_plgdata = 'https://data.plgrid.pl'
+hostname = 'ares'
+
+
+def fetch_bdo_files(json_data: dict) -> tuple[dict, int]:
+    """Function fetching result bdo files from cluster"""
+    session = requests.Session()
+    headers = {
+        "PROXY": json_data['grid_proxy']
+    }
+    job_id: str = json_data['job_id'].split('.')[0]
```

Comment:
Why do you need to split on `.`. Provide a comment with example format of this

Reply:
added comment